src/main/java/eventus/company/Company.java

```diff
@@ -15,7 +15,7 @@ public String getCompanyName(){
         return companyName;
     }
 
-    public String getContactNumber(){
+    public int getContactNumber(){
```

Comment:
Perhaps add a space between the closing parenthesis and the opening curly braces.